### ISSUE TITLE
A question concerning the IOU Loss

### ISSUE THREAD

**Issue by staeff777** [google/automl#699]
I'm trying to understand how the IOU loss is calculated, however, I am stuck with the question of how the respective prediction and target boxes are assigned to each other.

If my training dataset has 20 boxes and the Network finds 2 how are the matched?

**Comment by fsx950223** [comment]
You could reference https://github.com/tensorflow/addons/blob/dd11413736d3d3c854dc1f5fe3e9bd99ab7e3f83/tensorflow_addons/losses/iou_loss_test.py#L72

**Comment by staeff777** [comment]
Thank you. I'll try.
